core/domain/rule_domain_test.py

```diff
@@ -98,10 +100,47 @@ def test_that_all_rules_have_object_editor_templates(self):
                     self.assertTrue(
                         param_obj_type.has_editor_js_template(),
                         msg='(%s)' % clazz.description)
-                    at_least_one_rule_found = True
-                clses.append(clazz)
 
-        self.assertTrue(at_least_one_rule_found)
+        self.assertEqual(EXPECTED_TOTAL_NUMBER_OF_RULES, num_rules)
```

Comment:
Though, having said that -- it used to be just a check that num_rules was not zero, which probably conveys the point better. @wxyxinyu, @maitbayev -- what do you think?